README.md

```diff
@@ -39,3 +39,27 @@ jpm is required.
 - Update PATH: `export PATH="$HOME/node_modules/.bin/:$PATH"`
 
 Find more information on [MDN](https://developer.mozilla.org/en-US/Add-ons/SDK/Tools/jpm)
+
+
+Testing Checklist
+-----------------
+
+Installing
+
+- [ ] Extension installs 
+
+Popup
+
+- [ ] Popup opens
+
+- [ ] Search box is empty
+
+- [ ] Search works and defaults to DDG
+
+- [ ] Click a bang, the bang is added to the search box
+
+- [ ] Search with bang works
+
+- [ ] Expand/collapse button on right of search hides the bang options
+
+- [ ] Click "I'm feeling ducky" and search. Redirects to first organic
```

Comment:
I would also test:

-  saving the options, both for options.html and popup.html.
- safe search (you can just see if the URL has the param `&kp=-1` or you can use an IA like Urban Dictionary to verify)
- remember last search (enter a search in the popup, execute, try opening it later and see if last search is there)